pkgs/by-name/ry/ryujinx/package.nix

```diff
@@ -1,73 +1,89 @@
-{ lib
-, buildDotnetModule
-, dotnetCorePackages
-, fetchzip
-, libX11
-, libgdiplus
-, ffmpeg
-, openal
-, libsoundio
-, sndio
-, pulseaudio
-, vulkan-loader
-, glew
-, libGL
-, libICE
-, libSM
-, libXcursor
-, libXext
-, libXi
-, libXrandr
-, udev
-, SDL2
-, SDL2_mixer
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  darwin,
+  fetchFromGitHub,
+  libX11,
+  libgdiplus,
+  ffmpeg,
+  openal,
+  libsoundio,
+  sndio,
+  stdenv,
+  pulseaudio,
+  vulkan-loader,
+  glew,
+  libGL,
+  libICE,
+  libSM,
+  libXcursor,
+  libXext,
+  libXi,
+  libXrandr,
+  udev,
+  SDL2,
+  SDL2_mixer,
 }:
 
 buildDotnetModule rec {
   pname = "ryujinx";
-  version = "1.1.1401"; # Based off of the official github actions builds: https://github.com/Ryujinx/Ryujinx/actions/workflows/release.yml
+  version = "1.2.72";
 
-  src = fetchzip {
-    url = "https://archive.org/download/ryujinx-5dbba-07e-33e-83c-9047dcbb-701c-9655edbbe-89086.tar/Ryujinx-5dbba07e33e83c9047dcbb701c9655edbbe89086.tar.gz";
-    hash = "sha256-UeJ3KE5e5H9crqroAxjmxYTf/Z4cbj41a6/1HW2nLcA=";
+  src = fetchFromGitHub {
+    owner = "GreemDev";
+    repo = "Ryujinx";
+    rev = "master";
+    hash = "sha256-5lfqx4zoHw6b36e4rA5uh3bGmD1BLtGPjUMh3y4NfTg=";
   };
 
+  nativeBuildInputs = lib.optional stdenv.isDarwin [
+    darwin.sigtool
+    darwin.cctools
+  ];
+
   enableParallelBuilding = false;
 
   dotnet-sdk = dotnetCorePackages.sdk_8_0;
   dotnet-runtime = dotnetCorePackages.runtime_8_0;
 
   nugetDeps = ./deps.nix;
 
-  runtimeDeps = [
-    libX11
-    libgdiplus
-    SDL2_mixer
-    openal
-    libsoundio
-    sndio
-    pulseaudio
-    vulkan-loader
-    ffmpeg
-    udev
-
-    # Avalonia UI
-    glew
-    libICE
-    libSM
-    libXcursor
-    libXext
-    libXi
-    libXrandr
-
-    # Headless executable
-    libGL
-    SDL2
-  ];
+  runtimeDeps =
+    [
+      libX11
+      libgdiplus
+      SDL2_mixer
+      openal
+      libsoundio
+      sndio
+      vulkan-loader
+      ffmpeg
+
+      # Avalonia UI
+      glew
+      libICE
+      libSM
+      libXcursor
+      libXext
+      libXi
+      libXrandr
+
+      # Headless executable
+      libGL
+      SDL2
+    ]
+    ++ lib.optional (!stdenv.isDarwin) [
+      udev
+      pulseaudio
+    ]
+    ++ lib.optional stdenv.isDarwin [ darwin.moltenvk ];
```

Comment:
ofborg reports there is no `darwin.moltenvk`

Reply:
Oh yea. Seems like it got changed from darwin.moltenvk to moltenvk.

Reply:
Done